subsys/net/l2/openthread/openthread.c

```diff
@@ -279,6 +297,28 @@ enum net_verdict ieee802154_radio_handle_ack(struct net_if *iface,
 	return NET_CONTINUE;
 }
 
+void openthread_joiner_autostart(struct net_if *iface)
```

Comment:
One more issue spotted, this does not seem to build in default configuration, when joiner autostart is disabled:
```
/home/rolu/repos/zephyrproject/zephyr/subsys/net/l2/openthread/openthread.c:306:47: error: ‘CONFIG_OPENTHREAD_JOINER_PSKD’ undeclared (first use in this function)
   error = otJoinerStart(ot_context->instance, CONFIG_OPENTHREAD_JOINER_PSKD, NULL,
```

I guess we should conditionally disable building of this function with `#if defined(CONFIG_OPENTHREAD_JOINER_AUTOSTART)`

Reply:
Oh sorry, I've forgot it...
I fixed it.